src/layouts/Login/components/LoginForm.tsx

```diff
@@ -54,9 +54,7 @@ export const LoginForm = ({
             validate: validateEmail,
           })}
         />
-        {formState.errors.email && (
-          <FormErrorMessage>{formState.errors.email.message}</FormErrorMessage>
-        )}
+        <FormErrorMessage>{formState.errors.email?.message}</FormErrorMessage>
```

Comment:
Hey I am a bit confused about this, why did you remove this conditional ah? Doesn't it not make sense to only show `FormErrorMessage` iff there are errors

Reply:
this works because we are using chakra's form error message under the hood. at this [point](https://github.com/chakra-ui/chakra-ui/blob/main/packages/components/form-control/src/form-error.tsx#L37) in the source code, the form error message reads from the shared context (`FormControl`) and if it's invalid, then it renders. otherwise, it returns `null`, which is why this works auto-magically